defaults/aws_config.yaml

```diff
@@ -43,3 +43,8 @@ ami_id_db_scylla: ''
 ami_id_db_oracle: ''
 
 use_preinstalled_scylla: true
+
+aws_ebs_volume_num: 0
+aws_ebs_volume_type: "gp2"
+aws_ebs_volume_size: 500
+aws_ebs_volume_iops: 10000 # depend on type iops could be 100-16000 for io2|io3 and 3000-16000 for gp3
```

Comment:
@roydahan should we use the maximum available IOPS per device type?
Max allowed:
gp3: 16000 IOPS
io1, io2:  64000 IOPS
gp2: doesn't support setting IOPS

Reply:
The default should be like the default when setting this EBS on Amazon console UI.
Anyway, we should have the ability to control it from this parameter in case we will need to test with slow disk that we limit to certain amount of ops.